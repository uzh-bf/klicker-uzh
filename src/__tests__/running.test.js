/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import Running from '../pages/sessions/running'

describe('Snapshot-Testing', () => {
  it('Works', () => {
    const component = renderer.create(<Running />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
