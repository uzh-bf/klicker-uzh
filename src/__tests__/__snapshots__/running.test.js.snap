// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot-Testing Works 1`] = `
<div
  className="jsx-2966695483 jsx-140938897 commonLayout"
>
  <div
    className="jsx-179908404 teacherLayout"
  >
    <div
      className="jsx-179908404 navbar"
    >
      <div
        className="jsx-1373967784 navbar"
      >
        <div
          className="jsx-1373967784 sideArea"
        >
          <div
            className="ui borderless noBorder menu"
          >
            <a
              className="icon item sidebar"
              onClick={[Function]}
            >
              <i
                aria-hidden="true"
                className="sidebar icon"
              />
            </a>
            <h1
              className="jsx-1373967784"
            >
              runningSession.title
            </h1>
          </div>
        </div>
        <div
          className="jsx-1373967784 accountArea"
        >
          <div
            className="ui borderless loginArea noBorder menu"
          >
            <div
              className="right menu"
            >
              <div
                className="item"
                onClick={[Function]}
              >
                <a
                  className="jsx-1862355618"
                  href="/sessions/running"
                >
                  <button
                    className="ui small icon disabled left labeled button"
                    disabled={true}
                    onClick={[Function]}
                    role="button"
                    tabIndex={-1}
                  >
                    <i
                      aria-hidden="true"
                      className="play icon"
                    />
                    <span>
                      Running Session
                    </span>
                  </button>
                </a>
              </div>
              <a
                className="item"
                onBlur={[Function]}
                onClick={[Function]}
                onFocus={[Function]}
                onMouseEnter={[Function]}
                onMouseLeave={[Function]}
              >
                <i
                  aria-hidden="true"
                  className="info icon"
                />
                Updates
              </a>
              <div
                aria-busy={undefined}
                aria-disabled={true}
                aria-expanded={false}
                aria-multiselectable={undefined}
                className="ui disabled item simple dropdown"
                onBlur={[Function]}
                onChange={[Function]}
                onClick={[Function]}
                onFocus={[Function]}
                onMouseDown={[Function]}
                role="listbox"
                tabIndex={-1}
              >
                <div
                  aria-live="polite"
                  className="text"
                  role="alert"
                >
                  ANON
                </div>
                <i
                  aria-hidden="true"
                  className="user icon"
                  onClick={[Function]}
                />
                <div
                  className="menu transition"
                >
                  <div
                    aria-checked={undefined}
                    aria-disabled={undefined}
                    aria-selected={undefined}
                    className="item"
                    onClick={[Function]}
                    role="option"
                  >
                    <span>
                      Logout
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="jsx-179908404 content"
    >
      <div
        className="jsx-485800020 sidebar"
      >
        <div
          className="pushable"
        >
          <div
            className="ui vertical labeled icon ui overlay left wide sidebar sidebarMenu menu"
          >
            <a
              className="item"
              onClick={[Function]}
            >
              <span>
                Question Pool
              </span>
            </a>
            <a
              className="item"
              onClick={[Function]}
            >
              <span>
                Session List
              </span>
            </a>
            <a
              className="active item"
              onClick={[Function]}
            >
              <span>
                Running Session
              </span>
            </a>
            <a
              className="item"
              onClick={[Function]}
            >
              <span>
                Create Question
              </span>
            </a>
            <div
              className="jsx-485800020 langSwitcher"
            >
              <div
                className="languageSwitcher"
              >
                <div
                  aria-busy={undefined}
                  aria-disabled={undefined}
                  aria-expanded={false}
                  className="ui fluid search selection dropdown"
                  onBlur={[Function]}
                  onChange={[Function]}
                  onClick={[Function]}
                  onFocus={[Function]}
                  onMouseDown={[Function]}
                  role="combobox"
                  tabIndex={undefined}
                >
                  <input
                    aria-autocomplete="list"
                    autoComplete="off"
                    className="search"
                    onChange={[Function]}
                    style={
                      Object {
                        "width": undefined,
                      }
                    }
                    tabIndex={0}
                    type="text"
                    value=""
                  />
                  <div
                    aria-live="polite"
                    className="text"
                    role="alert"
                  >
                    English
                  </div>
                  <i
                    aria-hidden="true"
                    className="dropdown icon"
                    onClick={[Function]}
                  />
                  <div
                    aria-multiselectable={undefined}
                    className="menu transition"
                    role="listbox"
                  >
                    <div
                      aria-checked={true}
                      aria-disabled={undefined}
                      aria-selected={true}
                      className="active selected item"
                      onClick={[Function]}
                      role="option"
                      style={
                        Object {
                          "pointerEvents": "all",
                        }
                      }
                    >
                      <i
                        className="gb flag"
                      />
                      <span
                        className="text"
                      >
                        English
                      </span>
                    </div>
                    <div
                      aria-checked={false}
                      aria-disabled={undefined}
                      aria-selected={false}
                      className="item"
                      onClick={[Function]}
                      role="option"
                      style={
                        Object {
                          "pointerEvents": "all",
                        }
                      }
                    >
                      <i
                        className="de flag"
                      />
                      <span
                        className="text"
                      >
                        Deutsch
                      </span>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            className="pusher"
          >
            <div
              className="jsx-2801382701 messager"
            >
              <div
                className="ui message"
              >
                No currently running session...
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="jsx-179908404 actionArea"
    />
  </div>
</div>
`;
